src/components/ViewComments.jsx

```diff
@@ -1,26 +1,90 @@
-const ViewComments = ({ isLoadingComments, isReadingComments, comments }) => {
+import { useEffect, useState } from "react";
+import { postCommentToReview } from "../api";
+import { getCommentsOnReview } from "../api";
+
+const ViewComments = ({ isReadingComments, review_id }) => {
+  const [comment, setComment] = useState("");
+  const [comments, setComments] = useState([]);
+  const [isLoadingComments, setIsLoadingComments] = useState(false);
+  const [postError, setPostError] = useState(false);
+
+  useEffect(() => {
+    setIsLoadingComments(true);
+    getCommentsOnReview(review_id).then((data) => {
+      setComments(data.reverse());
```

Comment:
I can see why you've used the reverse function here based on where your add comment box is, this is fine but depending on how you want to implement the add comment you could have a button at the top of the comments to show/hide the add comment box.